4_compile.py

```diff
@@ -188,7 +188,7 @@ def configure_optimizers(self):
         callbacks=callbacks,
         accelerator="gpu",
         precision="16",
```

Comment:
Also, this is recommended in the 2.0 world

```suggestion
        precision="16-mixed",
```